.golangci.yml

```diff
@@ -19,8 +19,6 @@ linters-settings:
   importas:
     no-unaliased: true
     alias:
-      - pkg: connectrpc.com/connect
-        alias: connect
```

Comment:
Since v1.11.0 (the move to the connectrpc.com domain), it is no longer necessary/helpful to always have an import alias here. So I've removed this requirement and removed the (superfluous) alias in the file I touched below.